_data/schemas/compatibility.yaml

```diff
@@ -121,3 +121,58 @@ properties:
             caption:
               description: Caption to give the image
               type: string
+
+  segwit:
+    description: Container for information about segwit support
+    type: object
+    additionalProperties: false
+    properties:
+      tested: *tested
+      examples: *examples
+      features:
+        description: Segwit receiving features
+        type: object
+        properties:
+          receive:
+            description: Features related to generating segwit receive addresses
+            type: object
+            properties:
+              p2sh_wrapped: 
+                description: Does the wallet allow receiving to P2SH-wrapped segwit?
+                type: string
+                enum: *feat
+              bech32: 
+                description: Does the wallet allow receiving to Bech32 native segwit addresses?
+                type: string
+                enum: *feat
+              default: 
+                description: What is the default address receive type?
+                type: string
+                enum:
+                  - "p2pkh"
+                  - "p2sh"
+                  - "p2sh_wrapped"
+                  - "p2sh_wrapped_p2wsh"
+                  - "bech32"
+                  - "bech32_p2wsh"
+            additionalProperties: false
+          send:
+            description: Features related to sending to segwit addresses
+            type: object
+            properties:
+              bech32:
+                description: Does the wallet allow sending to Bech32 native segwit addresses?
+                type: string
+                enum: *feat
+              change_bech32:
+                description: Does the wallet return change to itself in a Bech32 native segwit address?
+                type: string
+                enum: *feat
+              segwit_v1:
+                description: Does the wallet allow sending to segwit v1 witnesses?
+                type: string (Does not pass validation, Bech32 not supported, Error during sending)
```

Comment:
I don't understand what "(Does not pass validation, Bech32 not supported, Error during sending)" is doing

Reply:
@jnewbery at some point I was considering making this an enum instead of free form string. Those were my ideas at the time. Will remove for now. Although I think having the field as enum isnt out of the question.